site/guide/model-documentation/working-with-model-documentation.qmd

```diff
@@ -66,7 +66,9 @@ This section describes how to work with model documentation in the {{< var valid
 
 ::::
 
-## Document Overview[^4]
+## Document Overview
```

Comment:
Needs sentence-style caps:

```suggestion
## Document overview
```

Reply:
My preference like above is to keep this in line with what's actually displayed in the UI, as this is a "named" page/feature.